File: hbdi/test/test3.php
<?php
error_log("test3.php loaded.");
$test = $_POST['ttt'];
$test2 = $_POST['test2'];
//echo "test: $test";
error_log("test should be here: $test", 0);
error_log("end of test3.php");

//
//$bar = isset($_POST['bar']) ? $_POST['bar'] : null;
//
//error_log($bar);
//error_log("visited");

?>

File: languages/java/test/src/Test3.java
package src;

class A {
	static String name = "A";

	static String greeting() {
		return "Class A";
	}
}

class B extends A {
	static String name = "B";
	static String greeting() {
		return "Class B";
	}
}

public class Test3 {
	public static void main(String[] args) {

		Test t = new Test();

		A a = new A();
		System.out.println(a.name);
		
		A b = new B();
		System.out.println(b.name);
		System.out.println(b.greeting());
	}

}
